README.md

```diff
@@ -175,16 +175,17 @@ Some Script Modules are so common that we've made them freely available in the [
 
 ### How `gruntwork-install` Works
 
-To actually install a Gruntwork Module, we wrote a bash script named `gruntwork-install`. Here's how it works:
+`gruntwork-install` helps you install a Gruntwork Module. Here's how it works:
 
 1. It uses [fetch](https://github.com/gruntwork-io/fetch) to download the specified version of the scripts or binary from
    the (public or private) git repo specified via the `--repo` option.
-1. If you used the `--module-name` parameter, it downloads the files from the `modules` folder of `--repo` and runs
-   the `install.sh` script of that module.
-1. If you used the `--binary-name` parameter, it downloads the right binary for your OS, copies it to `/usr/local/bin`,
-   and gives it execute permissions.
+   1. You need to specify either a module name or a binary name. 
+       - If you use the `--module-name` parameter, it downloads the files from the `modules` folder of `--repo` and runs
+          the `install.sh` script of that module.
+              - If you use the `--binary-name` parameter, it downloads the right binary for your OS, copies it to `/usr/local/bin`,
+                 and gives it execute permissions.
 
-That's it!
+                 That's it!
```

Comment:
This rendering on this change doesn't quite look right.

<img width="1120" alt="Screen Shot 2022-08-23 at 12 20 45 PM" src="https://user-images.githubusercontent.com/430092/186223950-13b3c5cc-2f94-47bb-aa4e-e738f73a5804.png">

Maybe remove it altogether since I don't think that `That's it` really adds to the docs?